SPECS/tdnf/tdnf.spec

```diff
@@ -56,7 +51,6 @@ Requires:       rpm-libs
 Requires:       tdnf-cli-libs = %{version}-%{release}
 Requires:       zlib
 Obsoletes:      yum
```

Comment:
If you're already removing the `Provides: yum` below, we might as well remove this too. It would be even more "correct", because we never had `yum`, so there's really nothing to obsolete.:)